images-list

```diff
@@ -998,7 +1002,9 @@ quay.io/kiwigrid/k8s-sidecar rancher/mirrored-kiwigrid-k8s-sidecar 1.12.3
 quay.io/kiwigrid/k8s-sidecar rancher/mirrored-kiwigrid-k8s-sidecar 1.15.6
 quay.io/kiwigrid/k8s-sidecar rancher/mirrored-kiwigrid-k8s-sidecar 1.15.9
 quay.io/kiwigrid/k8s-sidecar rancher/mirrored-kiwigrid-k8s-sidecar 1.19.2
+quay.io/minio/mc rancher/mirrored-minio-mc RELEASE.2022-12-13T00-23-28Z
```

Comment:
`minio/mc` is present in this file already, being pulled from Docker Hub. Does this specifically need to be pulled from Quay?

Reply:
`quay.io` was the registry I saw used in the E helm chart ... Testing `d pull` locally without `quay.io` I confirm that the images are in docker hub as well. ... And checking the images-list further I see that a `minio/minio` section exists too.

I will move all the `quay.io/minio/*` entries to plain `minio`.

Reply:
Done with commit 7b8799690e22086d496a232beebb77688e7e49ff